app/assets/stylesheets/components.scss

```diff
@@ -7,7 +7,7 @@
 
 .dgu-highlight {
   background: $grey-3;
```

Comment:
Probably worth updating your colours while you're in here, eg govuk-colour("light-grey")